datasets/dataset_imdb.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+IMDB dataset.
+inspired from:
+https://keras.io/examples/nlp/text_classification_from_scratch/
+https://builtin.com/data-science/how-build-neural-network-keras
+"""
+
+import tensorflow as tf
+import numpy as np
+from tensorflow import keras
+from keras.utils import to_categorical
+from keras import models
+from keras import layers
```

Comment:
You may want to use a single line for this : 
from keras import models, layers

Reply:
Done, the imports seem clean:
https://github.com/SubstraFoundation/distributed-learning-contributivity/blob/a0d52796ba6cf1519947a4b2b70fa5e022ec1dd0/datasets/dataset_imdb.py#L9-L13